Scripts/Tracking/Follow/Modifier/ModifyRigidbodyVelocity.cs

```diff
@@ -0,0 +1,105 @@
+
+        /// <summary>
+        /// Moves the source <see cref="Rigidbody"/> by applying velocity so it tracks the to the target <see cref="Transform.position"/>.
+        /// </summary>
+        /// <param name="source">The source <see cref="Transform"/> to modify.</param>
+        /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>
+        public override void UpdatePosition(Transform source, Transform target)
+        {
+            CachedSource = source;
+            CachedTarget = target;
+            if (source != null && target != null && SourceRigidbody != null)
+            {
+                Vector3 positionDelta = target.position - source.position;
+                Vector3 velocityTarget = positionDelta / Time.fixedDeltaTime;
+                Vector3 calculatedVelocity = Vector3.MoveTowards(SourceRigidbody.velocity, velocityTarget, maxDistanceDelta);
+
+                if (velocityLimit == float.PositiveInfinity || calculatedVelocity.sqrMagnitude < velocityLimit)
+                {
+                    SourceRigidbody.velocity = calculatedVelocity;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Rotates the source <see cref="Rigidbody"/> by applying angular velocity so it tracks the to the target <see cref="Transform.rotation"/>.
+        /// </summary>
+        /// <param name="source">The source <see cref="Transform"/> to modify.</param>
+        /// <param name="target">The target <see cref="Transform"/> to utilize in the modification.</param>
+        public override void UpdateRotation(Transform source, Transform target)
+        {
+            CachedSource = source;
+            CachedTarget = target;
+            if (source != null && target != null && SourceRigidbody != null)
+            {
+                Quaternion rotationDelta = target.rotation * Quaternion.Inverse(source.rotation);
+
+                float angle;
+                Vector3 axis;
+                rotationDelta.ToAngleAxis(out angle, out axis);
+
+                angle = ((angle > 180) ? angle -= 360 : angle);
+
+                if (angle != 0)
+                {
+                    Vector3 angularTarget = angle * axis;
+                    Vector3 calculatedAngularVelocity = Vector3.MoveTowards(SourceRigidbody.angularVelocity, angularTarget, maxDistanceDelta);
+                    if (angularVelocityLimit == float.PositiveInfinity || calculatedAngularVelocity.sqrMagnitude < angularVelocityLimit)
+                    {
+                        SourceRigidbody.angularVelocity = calculatedAngularVelocity;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Does not perform any modification
+        /// </summary>
+        /// <param name="source">Unused.</param>
+        /// <param name="target">Unused.</param>
+        public override void UpdateScale(Transform source, Transform target)

```

Comment:
scale is an odd one. But i do kinda think that this is more appropriate where you explicitly state that scaling isn't going to work with this modifier. Having it part of the superclass means it will always scale and I'd argue you actually dont want any scaling here

Reply:
I wonder if we should offer an enum that sits on the base class, that way the user can decide what to sync with a specific modifier. I guess that means we'd support having multiple ones which makes it more complex, though. 😞